contrib/hbwin/hbwin.hbp

```diff
@@ -29,7 +29,7 @@ mapi.c
 oemansi.c
 olecore.c
 oleuuid.c{watcom}
-hbolesrv.c
+{HBWIN_HAS_OLESRV}hbolesrv.c
```

Comment:
I don't know what is `hbwin32` sorry, there is no such library in Harbour sources.

Yes, exactly. You copied this line, but you didn't build a dynamic version of this lib, so if you request to include the dynamic version of it, it will naturally fail. Either build the dynamic version of it, or remove the PREF/SUFF decorations so it will always link the static lib regardless of the content your `_HB_DYNSUFF` variable.